### PR TITLE
fix: expose the configuration of HTTP request node to Docker

### LINKED ISSUE #8713: HTTP Request: Text size is too large, max size is 1.00 MB, but current size is 1.69 MB.
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.8,3

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

image

I noticed that there is an HTTP_REQUEST_NODE_MAX_TEXT_SIZE environment variable when starting with source code, but this variable does not exist when starting with Docker.

Please synchronize this environment variable to the docker deployment method.

### ✔️ Expected Behavior

_No response_

### ❌ Actual Behavior

_No response_

### PR DESCRIPTION
**Pull request by zhuiyue132**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] Please open an issue before creating a PR or link to an existing issue
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

Fixes https://github.com/langgenius/dify/issues/8713

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

This PR fixes https://github.com/langgenius/dify/issues/8713

### CODE REVIEW

**Comment by crazywoola** [comment]
Please link related issue and update the docs here accordingly https://github.com/langgenius/dify-docs

**Comment by zhuiyue132** [comment]
This PR fixes #8713

File: docker/.env.example [hunk 1]
```diff
@@ -568,6 +568,10 @@ WORKFLOW_MAX_EXECUTION_STEPS=500
 WORKFLOW_MAX_EXECUTION_TIME=1200
 WORKFLOW_CALL_MAX_DEPTH=5
 
+# HTTP request node in workflow configuration
+HTTP_REQUEST_NODE_MAX_BINARY_SIZE=10485760 # 10MB
```
Comment (crazywoola) [inline docker/.env.example]:
```suggestion
HTTP_REQUEST_NODE_MAX_BINARY_SIZE=10485760
```

File: docker/.env.example [hunk 2]
```diff
@@ -568,6 +568,10 @@ WORKFLOW_MAX_EXECUTION_STEPS=500
...
+HTTP_REQUEST_NODE_MAX_TEXT_SIZE=1048576 # 1MB
```
Comment (crazywoola) [inline docker/.env.example]:
```suggestion
HTTP_REQUEST_NODE_MAX_TEXT_SIZE=1048576
```

**Review by crazywoola — DISMISSED** [review]

**Comment by zhuiyue132** [comment]
> Please link related issue and update the docs here accordingly https://github.com/langgenius/dify-docs

done.  
[docs pr](https://github.com/langgenius/dify-docs/pull/282)

**Review by crazywoola — APPROVED** [review]

**Pull request merged**